__tests__/GitUtils-test.js

```diff
@@ -0,0 +1,17 @@
+import GitUtils from '../lib/GitUtils';
+
+describe('GitUtils', () => {
```

Comment:
It would be great to see some unit tests that mock the output of the `exec('git log ...');` so you can test how the matching works with the actual output of a `git log` command.

So you could actually run that command for a couple different repos in your terminal, and then copy the output and use it as a mocked response for the `exec` command. That way you can test `getPullRequestsMergedBetween` in addition to `getPullRequestNumberFromCommitMessage`.

Reply:
Sure, I added test for getPullRequestsMergedBetween by mocking exec.